### PR TITLE
Put WebLogic operator into the Istio Mesh

### PR DESCRIPTION
**Pull request by pfmackin**
This PR has changes to put the WebLogic Operator in the Istio mesh and configure it to have access to WebLogic applications in the mesh.  This is needed so that the operator can make admin REST calls to the WebLogic applications in the mesh.  When this is working correctly, the domain status is correctly updated, for example:

    servers:
    - desiredState: RUNNING
      health:
        activationTime: "2021-05-18T18:23:49.440000Z"
        overallHealth: ok

When it was NOT working (the bug), the overallHealth is [not available]

The following changes were made:
1. Remove weblogic-operator from the istio sidecar config neverInjectSelector. 
2. Add the excludeOutbound port annotation to the weblogic operator so that it can access the API server
3. Add the WebLogic operator service account to the list of principals in istio AuthorizationPolicy for the application.
4. Fix the NetworkPolicies so that the weblogic operator can talk to the istio control plane

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/thirdparty/charts/weblogic-operator/templates/_operator-dep.tpl [hunk 1]
```diff
@@ -22,6 +22,10 @@ spec:
      labels:
         weblogic.operatorName: {{ .Release.Namespace | quote }}
         app: "weblogic-operator"
+     annotations:
```
Comment (jmaron99) [inline platform-operator/thirdparty/charts/weblogic-operator/templates/_operator-dep.tpl:25]: Have we filed a JIRA with WebLogic operator?

File: platform-operator/helm_config/charts/verrazzano/templates/08-thirdparty-networkpolicy.yaml [hunk 2]
```diff
@@ -15,6 +15,11 @@ spec:
       app: weblogic-operator
   policyTypes:
     - Ingress
+  ingress:
```
Comment (jmaron99) [inline platform-operator/helm_config/charts/verrazzano/templates/08-thirdparty-networkpolicy.yaml:18]: You didn't find you had to allow egress to CoreDNS (lookup Istiod) and Istiod?

File: platform-operator/thirdparty/charts/weblogic-operator/templates/_operator-dep.tpl (hunk 1 above)
Comment (pfmackin) [reply platform-operator/thirdparty/charts/weblogic-operator/templates/_operator-dep.tpl:25]: not yet, on my list today to sync up with Derek

File: platform-operator/helm_config/charts/verrazzano/templates/08-thirdparty-networkpolicy.yaml (hunk 2 above)
Comment (pfmackin) [reply platform-operator/helm_config/charts/verrazzano/templates/08-thirdparty-networkpolicy.yaml:18]: There are no egress restrictions, I removed them all across the board the other day

Comment (jmaron99) [reply platform-operator/helm_config/charts/verrazzano/templates/08-thirdparty-networkpolicy.yaml:18]: Was looking at an old version locally that still had egress defined

**Review by jmaron99 — APPROVED** [review]
LGTM

**Pull request merged**